LocalMultiplayerAgent/LocalMultiplayerAgentPublishFolder/appsettings.Development.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "Logging": {
+    "IncludeScopes": false,
+    "LogLevel": {
+      "Default": "Debug",
```

Comment:
files in this folder should not be published. maybe add this file to .gitignore?